.copr/Makefile

```diff
@@ -1,5 +1,5 @@
 srpm:
-	./build.sh srpm
+	./build.sh srpm --with-timestamp --with-commit-id
```

Comment:
please change it as:

`./build.sh --with-timestamp --with-commit-id srpm`

Just noticed the issue. :man_facepalming:

Reply:
I wonder if it'd be possible to remove the dependency on `git`... I already had to patch `v4.4.x`'s `build.sh` to actually support building on RHEL 7 (since `git` there doesn't understand `git -C`...).